src/Lumi/Components/Form.purs

```diff
@@ -226,10 +226,7 @@ useForm
   :: forall props unvalidated result
    . Mapping ModifyValidated unvalidated unvalidated
   => FormBuilder
-       { initialState :: unvalidated
-       , readonly :: Boolean
-       , inlineTable :: Boolean
-       , forceTopLabels :: Boolean
```

Comment:
These props are specific to the renderer and, thus, should not propagate to the form itself; keeping those could break a lot of code if we were to use these hooks.